crates/bevy_pbr/src/light_probe/environment_map.rs

```diff
@@ -137,6 +137,10 @@ pub struct EnvironmentMapIds {
 /// surrounding a region in space. For more information, see
 /// [`crate::environment_map`].
 #[derive(Bundle, Clone)]
+#[deprecated(
+    since = "0.15.0",
+    note = "Use the `LightProbe` and `EnvironmentMapLight` components instead. Inserting them will now also insert the other components required by them automatically."
+)]
```

Comment:
Question: Do we actually want to deprecate this yet, considering you still need to add both components? Or should I only remove the `SpatialBundle` from here?

Reply:
Definitely unfortunate that the `EnvironmentMapLight` component is also used on camera entities. Looking at the examples it feels like that should be merged into the `Skybox` component. We could then require `LightProbe` for `EnvironmentMapLight`, but alas.

I think we should deprecate this, even though it's not perfect

Reply:
What do you mean by "also used on camera entities".  Where else other than camera entities is it used?